### PR TITLE
Fix command count check, commenting cleanup

### PR DESCRIPTION
**Pull request by itsmojo**

### CODE REVIEW

File: OmniKit/MessageTransport/MessageBlocks/GetStatusCommand.swift [hunk 1]
```diff
@@ -9,29 +9,29 @@
...
-//    }
-    
+    // https://github.com/openaps/openomni/wiki/Command-0E-Status-Request
+
     public let blockType: MessageBlockType = .getStatus
     public let length: UInt8 = 1
     public let podInfoType: PodInfoResponseSubType
+
+//    public enum PodInfoResponseSubType: UInt8, Equatable {
+//        case normal                      = 0x00
```
Comment (ps2) [inline OmniKit/MessageTransport/MessageBlocks/GetStatusCommand.swift]: Having this here is likely to get out of date with the actual definition in PodInfo.swift

**Review by ps2 — COMMENTED** [review]
Bugfix looks good. I'm not sure that putting links to the wiki in the source code (other than maybe a high level link) is a great idea; the wiki is likely to change, and then the comments will rot.

File: OmniKit/MessageTransport/MessageBlocks/GetStatusCommand.swift (hunk 1 above)
Comment (itsmojo) [reply OmniKit/MessageTransport/MessageBlocks/GetStatusCommand.swift]: Just trying to update stuff that was already there. I'm happy to just toss it.

**Comment by itsmojo** [comment]
I was just updating wiki references that I noticed were out of date due to command name changes and tried to make them more helpful for all the 02 response subtypes. Per Zulip conversation will remove all the wiki references for individual commands, but add one to MessageBlock.swift to reference master Message-Types wiki page

**Comment by ps2** [comment]
Tests are failing after this commit:
OmniKitTests.PodInfoTests
  testPodInfoTester, failed - message decoding threw error: notEnoughData
  /Users/travis/build/ps2/rileylink_ios/OmniKitTests/PodInfoTests.swift:322
  ```
        } catch (let error) {
            XCTFail("message decoding threw error: \(error)")
        }
  ```

**Pull request merged**